### ISSUE TITLE
bigip_sys_snmp_traps port 514 breaks state

### ISSUE THREAD

**Issue by oniGino** [F5Networks/terraform-provider-bigip#448]
provider version 1.7.0
ltm version 14.1.4

````
resource "bigip_sys_snmp_traps" "snmp_traps" {
  name        = "/Common/nms"
  community   = "ro"
  host        = "xxx"
  description = "vnms1"
  port        = 514
}
```

```
  # .bigip_sys_snmp_traps.snmp_traps will be updated in-place
  ~ resource "bigip_sys_snmp_traps" "snmp_traps" {
      + community   = "XXX"
      + description = "XXX"
      + host        = "1XXX"
        id          = "XXX"
      + name        = "XXX"
        # (1 unchanged attribute hidden)
    }

```
 
So if you declare a trap object with port 514, on the next apply
terraform will try to change that port to 0 and will not keep state properly

the object basically reports changes needed on every apply

if you do say yes to the apply the operation will fail

```
Error: HTTP 403 :: {"code":403,"message":"Operation is not supported on property /sys/snmp/traps.","errorStack":[],"apiError":1}
```

**Comment by nmenant** [comment]
I've been able to reproduce the issue. Tracking this internally with INFRAANO-291

**Comment by trinaths** [comment]
Issue resolved with https://github.com/F5Networks/terraform-provider-bigip/pull/473/files.

**Comment by RavinderReddyF5** [comment]
@oniGino fixed in v1.9.0, please verify
